aocd/__init__.pyi

```diff
@@ -0,0 +1,35 @@
+import typing as t
+
+from aocd import cli, cookies, exceptions, get, models, post, runner, transforms, utils, version
+from aocd.exceptions import AocdError, PuzzleUnsolvedError
+from aocd.get import get_data
+from aocd.post import submit
+from aocd.utils import AOC_TZ
+from aocd.version import __version__
+
+# TODO: does this work as intended?
```

Comment:
I'm not sure how `__all__` works in `pyi` files. I need to make sure this'll work as intended.